README.md

```diff
@@ -157,23 +157,19 @@ This example:
 
 If there is `dump.pkl` inside `/actual/dumpdir`, you can load it with:
 ```
-docker run -it \
+docker run \
   -p 127.0.0.1:5000:5000 \
   --mount type=bind,source=/actual/dumpdir,target=/dumpdir \
   shardlabs/starknet-devnet \
-  poetry run starknet-devnet \
-  --host 0.0.0.0 --port 5000 \
```

Comment:
Since this is changed, can you also change line 156 so that it doesn't mention the second sentence:
> However, leave the `--host 0.0.0.0` part as it is.